.github/workflows/nightly.yml

```diff
@@ -89,11 +89,19 @@ jobs:
             python3-jsonschema \
             xsltproc \
             zlib1g-dev \
-            appstream;
+            appstream
+      - name: Get architecture
+        run: |
+          echo "ARCHITECTURE=$(uname -m)" >> $GITHUB_ENV
+      - name: Add ppa for newer multimedia libs
+        if: ${{ env.ARCHITECTURE == 'x86_64' }}
```

Comment:
It would be much better to build native packages on OBS instead of providing AppImages built with libraries that are years old.